lib/internal/streams/writable.js

```diff
@@ -75,14 +75,6 @@ function nop() {}
 const kOnFinished = Symbol('kOnFinished');
 
 function WritableState(options, stream, isDuplex) {
-  // Duplex streams are both readable and writable, but share
-  // the same options object.
-  // However, some cases require setting options to different
-  // values for the readable and the writable sides of the duplex stream,
-  // e.g. options.readableObjectMode vs. options.writableObjectMode, etc.
-  if (typeof isDuplex !== 'boolean')
-    isDuplex = stream instanceof Stream.Duplex;
```

Comment:
I think this is breaking

Reply:
It is not, because WritableState is only called in one place, and isDuplex parameter is already a boolean. As the comment states for the deleted lines, I believe this is a dead code.

Reply:
`WritableState` is exported... so it can be instantiated by user code...

Reply:
@mcollina wdyt?

Reply:
I would ship it in v19 and tag it as "backing for LTS" out of caution.